tests/request_data/onboarding.json

```diff
@@ -1,14 +1,7 @@
 {
   "interest_profile": {
     "profile_id": "b5e9b183-3264-4e08-aff2-cbc6bd0583b3",
-    "click_history": [
-      {
-        "article_id": "2169d95d-8691-479e-8ac6-1bc86e33ed2e"
-      },
-      {
-        "article_id": "a713c6f1-6b09-4730-9c83-585f29ec475a"
-      }
-    ],
+    "click_history": [],
```

Comment:
Looks like these changes from taste-testing the recs slipped into the PR, but I don't think we need to change this file